bigquery_etl/glam/generate.py

```diff
@@ -122,25 +122,28 @@ def main():
             **models.clients_histogram_aggregates(parameterize=True),
         ),
         table(
-            "scalar_bucket_counts_v1",
-            **models.scalar_bucket_counts(
+            "clients_scalar_bucket_counts_v1",
+            **models.clients_scalar_bucket_counts(
```

Comment:
This reverts the name of the table since the query is now granular at the client id level.

Reply:
Actually, it looks like I misunderstood this. Client ids should not show up in the resulting table, but only during cubing.